.github/workflows/build-images.yml

```diff
@@ -0,0 +1,95 @@
+name: Build images
+
+on:
+  - push
+
+jobs:
+  cc7_lcg95:
+    runs-on: ubuntu-latest
+    needs: base_images
+    steps:
+      - uses: actions/checkout@v1
+      - name: Extract branch name
+        shell: bash
+        run: echo "::set-env name=branch::$(echo ${GITHUB_REF#refs/heads/} | sed 's/\//_/g')"
+      - uses: acts-trk/Publish-Docker-Github-Action@master
```

Comment:
This should be `acts-project/...`, right?

Reply:
yes